agnostic-astro/package.json

```diff
@@ -10,11 +10,38 @@
     "url": "https://github.com/AgnosticUI/agnosticui.git"
   },
   "exports": {
-    ".": "./src/lib/components/index.js"
+    ".": "./src/lib/components/index.js",
+    "./Alert": "./src/lib/components/Alert.astro",
```

Comment:
This was a bit annoying but I just wasn't able to glob all these in a way that produced a nice import experience 🤷‍♂️

Reply:
https://github.com/AgnosticUI/agnosticui/issues/208 needs to be done here too I think